lang/c++/include/avro/DataFile.hh

```diff
@@ -39,7 +39,11 @@ enum Codec {
     DEFLATE_CODEC,
 
 #ifdef SNAPPY_CODEC_AVAILABLE
-    SNAPPY_CODEC
+    SNAPPY_CODEC,
+#endif
+
+#ifdef ZSTD_CODEC_AVAILABLE
+    ZSTD_CODEC
```

Comment:
It seems to me that these enum values should really have an explicit value to avoid potential confusion where "2" could mean either depending on what preprocessor definitions are set.

Reply:
Agreed. Added explicit values.